eclipse-scout-core/src/lookup/RestLookupCall.js

```diff
@@ -8,9 +8,38 @@
  * Contributors:
  *     BSI Business Systems Integration AG - initial API and implementation
  */
-import {arrays, LookupCall, scout} from '../index';
+import {arrays, LookupCall, objects, scout} from '../index';
 import $ from 'jquery';
 
+/**
+ * A lookup call that can load lookup rows from a REST service.
+ *
+ * API:
+ * ----
+ * By default, the REST service is expected to listen for POST requests at the URL defined by
+ * this.resourceUrl. It receives a restriction object and must return a list of matching lookup rows.
+ * The serialization format is JSON.
+ *
+ * Lookup rows:
+ * ------------
+ * The standard lookup row properties defined by Scout are usually sufficient (see AbstractLookupRowDo.java).
+ *
+ * Restriction:
+ * ------------
+ * The restriction object consists of a number of 'well-known' properties (e.g. 'text' in QueryBy.TEXT
+ * mode, see AbstractLookupRestrictionDo.java for details) and additional, service-dependent properties
+ * that can either be predefined in the model or added programmatically at runtime. Since all of those
+ * properties are sent in the same restriction object, some care must be taken to prevent accidental
+ * overwriting of properties.
+ *
+ * Order of precedence (lowest to highest):
+ * 1. Restrictions automatically applied to all clones after their creation.
+ *    These are: 'active' (ALL, TEXT, REC) and 'maxRowCount' (ALL, TEXT, REC)
```

Comment:
maybe add:  
"1. Restrictions automatically applied to all clones after their creation **by their corresponding cloneFor method**"

Reply:
OK